content/community/teams.md

```diff
@@ -0,0 +1,59 @@
+---
+prev: /community/
+next: /community/rules/
+---
+
+# Teams and Organization
```

Comment:
After reviewing, I think it's better to:

- rename this file to `content/community/moderation.md`
- change the title to `# Moderation`
- summarize how we moderate, briefly introduce roles (`admin` and `mod`)

Then

- further explain admins
  - maybe include some history and mention the Qualified team (can be done later)
- further explain mods
- explain how to contact us